tests/test_mapping_utils.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from gdcmodels import mapping_utils
+
```

Comment:
What happens in this case?

```python
one = {"a": [1,2]}
two = {"a": [1,3]}
```

But I don't know if we have properties that are list-like.

Reply:
This is a good observation because it will join the two even if there are what we would consider duplicates. I think this behavior is fine because we don't have this situation in the [mapping files](https://github.com/NCI-GDC/gdc-models/blob/develop/es-models/gdc_from_graph).